packages/instrumentation-node-cache/package.json

```diff
@@ -0,0 +1,72 @@
+{
+    "name": "opentelemetry-instrumentation-node-cache",
+    "version": "0.22.0",
+    "description": "open telemetry instrumentation for the `node-cache`",
+    "keywords": [
+        "node-cache",
+        "opentelemetry"
+    ],
+    "author": {
+        "name": "Aspecto",
+        "email": "support@aspecto.io",
+        "url": "https://aspecto.io"
+    },
+    "homepage": "https://github.com/aspecto-io/opentelemetry-ext-js",
+    "license": "Apache-2.0",
+    "main": "dist/src/index.js",
+    "files": [
+        "dist/**/*.js",
+        "dist/**/*.js.map",
+        "dist/**/*.d.ts",
+        "LICENSE",
+        "README.md"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aspecto-io/opentelemetry-ext-js.git"
+    },
+    "scripts": {
+        "build": "tsc",
+        "prepare": "yarn run build",
+        "test": "mocha --require opentelemetry-instrumentation-testing-utils",
+        "test:jaeger": "OTEL_EXPORTER_JAEGER_AGENT_HOST=localhost mocha --require opentelemetry-instrumentation-testing-utils",
+        "test:ci": "tav",
+        "watch": "tsc -w",
+        "version:update": "node ../../scripts/version-update.js",
+        "version": "yarn run version:update"
+    },
+    "bugs": {
+        "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
+    },
+    "peerDependencies": {
+        "@opentelemetry/api": "^1.0.0"
+    },
+    "dependencies": {
+        "@opentelemetry/core": "^0.22.0",
+        "@opentelemetry/instrumentation": "^0.22.0",
+        "@opentelemetry/semantic-conventions": "^0.22.0"
+    },
+    "devDependencies": {
+        "@opentelemetry/api": "^1.0.0",
+        "@opentelemetry/tracing": "^0.22.0",
+        "@types/mocha": "^8.2.2",
+        "expect": "^26.6.2",
+        "mocha": "^8.4.0",
+        "node-cache": "^5.1.2",
```

Comment:
Since we support 5.0.0 and above, shouldn't it be also 5.0.0 ?

Reply:
I think it's ok to pick one specific version for development purposes.
In the ci we have test-all-versions which checks all the `>=5.0.0` versions